unit_tests/test_redis_client_base.py

```diff
@@ -0,0 +1,179 @@
+import json
+import uuid
+import pytest
+from redis import StrictRedis
+import d3a_api_client
+from d3a_api_client.redis_client_base import RedisClientBase, RedisAPIException
+from unittest.mock import MagicMock
+
+area_id = str(uuid.uuid4())
+transactionID = str(uuid.uuid4())
+d3a_api_client.redis_client_base.StrictRedis = MagicMock(spec=StrictRedis)
+d3a_api_client.redis_client_base.StrictRedis.publish = MagicMock(spec=StrictRedis)
+d3a_api_client.redis_client_base.StrictRedis.pubsub.psubscribe = MagicMock()
+
+
+@pytest.fixture
+def redis_client_auto_register():
+    return RedisClientBase(area_id=area_id, autoregister=False)
+
+
+class TestRedisClientBase:
+
+    def test_function__subscribe_to_response_channels(self, redis_client_auto_register):
+        RedisClientBase._subscribe_to_response_channels = MagicMock()
+        redis_client_auto_register._subscribe_to_response_channels()
+        redis_client_auto_register.pubsub.psubscribe.assert_called_with(
+            **{f"{area_id}/response/register_participant": redis_client_auto_register._on_register,
+               f"{area_id}/response/unregister_participant": redis_client_auto_register._on_unregister,
+               f"{area_id}/*": redis_client_auto_register._on_event_or_response,
+               "aggregator_response": redis_client_auto_register._aggregator_response_callback})
+
+    def test_function_aggregator_response_callback(self, redis_client_auto_register):
+        message = {'data': {"transaction_id": f"{transactionID}"}}
+        message["data"] = json.dumps(message['data'])
+        redis_client = redis_client_auto_register
+        redis_client._transaction_id_buffer.append(transactionID)
+        redis_client._aggregator_response_callback(message)
+        assert redis_client._transaction_id_buffer == []
+
+    def test_function_check_buffer_message_matching_command(self, redis_client_auto_register):
```

Comment:
```suggestion
    def test_check_buffer_message_matching_command_and_id(self, redis_client_auto_register):
```
The name of the tested method wasn't complete